dwh/query_lib_biq_query_test.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import query_lib_big_query as ql
+"""
+For these tests to run the Google Auth settings have to be run on the machine running these tests:
```

Comment:
Let's keep these as unit-tests, i.e., they should not make a network call and should not require a GCP project. You can validate the SQL queries instead of returned column values.

We also want to test the actual BigQuery connection/querying but that should go into integration tests, similar to `validate_indicators.sh`. Can you please start from there and adapt it for a BigQuery version too; we should get the same results as the Spark version.

Reply:
I moved these tests into `validate_indicators.sh`.  Since we have integration tests exercising the queries and also returned data, I don't see a need to have another set of tests validating the query string.